.github/workflows/device-tests-android.yml

```diff
@@ -39,7 +39,7 @@ jobs:
         with:
           name: device-test-android
           if-no-files-found: error
-          path: test/Sentry.Maui.Device.TestApp/bin/Release/net8.0-android/android-x64/io.sentry.dotnet.maui.device.testapp-Signed.apk
+          path: test/Sentry.Maui.Device.TestApp/bin/Release/net8.0-android34.0/android-x64/io.sentry.dotnet.maui.device.testapp-Signed.apk
```

Comment:
if you use -o when building to set the path, you can have a directory that's not going to change and avoid having to keep up with tfm changes

Reply:
Ah, good idea... next time we touch this we can do that instead.